README.md

```diff
@@ -158,55 +158,37 @@ Aleth supports EVMC out of the box, but geth not yet.
 	> cmake --build .
 	```
 
-1. Download the [genesis file](ewasm-testnet-geth-config.json) and use it to initialize geth:
-
-	```sh
-	> wget https://raw.githubusercontent.com/ewasm/testnet/master/ewasm-testnet-geth-config.json
-	> ./build/bin/geth --datadir /tmp/ewasm-node/4201/ init ewasm-testnet-geth-config.json
-	```
-
-	Note that the `/tmp/ewasm-node/4201` directory name above is arbitrary. It just needs to be unique.
-
-1. Run geth with hera and connect to the testnet:
+1. Run geth with hera:
 
 	Use `--vm.ewasm` flag in `geth` to plug in an EVMC VM shared library. Point it to the Hera shared library that you built a moment ago. 
 	Additional EVMC options can be provided after a comma. 
 	Hera options are documented in [hera](https://github.com/ewasm/hera).
 
-	Note also the `--etherbase`, `--networkid`, and `--bootnodes` commands, below, and copy them verbatim as these are required to connect to and sync with the Ewasm testnet.
-
 	The `--vmodule` argument sets the verbosity for the `eth` and `p2p` modules, which will provide lots of juicy debugging information on your node's connection to the other testnet peers, and on its mining, accepting, and propagating blocks. Feel free to reduce verbosity or turn this off.
 
-	Finally, if you want your node to participate in mining, add the arguments `--mine --miner.threads 1`.
+	Finally, if you want your node to mine, add the arguments `--mine --miner.threads 1`.
 
 	Check out the geth [CLI wiki page](https://github.com/ethereum/go-ethereum/wiki/Command-Line-Options) for more information on these commands, or just run `geth --help`.
 
-	Here's the recommended configuration for connecting your node to the Ewasm testnet:
+	Here's the recommended configuration for running a node locally:
 
 	```sh
 	> ./build/bin/geth \
 	--vm.ewasm="/path/to/libhera.so,metering=true,fallback=true" \
 	--datadir /tmp/ewasm-node/4201/ \
-	--etherbase 031159dF845ADe415202e6DA299223cb640B9DB0 \
 	--rpc --rpcapi "web3,net,eth,debug" \
-	--rpcvhosts="*" --rpcaddr "0.0.0.0" \
+	--rpcvhosts=* --rpcaddr "0.0.0.0" \
```

Comment:
I think this needs to stay in quotes otherwise the shell expands it.

Reply:
Fixed